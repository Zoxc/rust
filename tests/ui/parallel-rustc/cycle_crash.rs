//@ compile-flags: -Z threads=2
//~^ERROR cycle detected when

const FOO: usize = FOO;

fn main() {}
