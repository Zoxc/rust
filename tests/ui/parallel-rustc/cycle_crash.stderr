error[E0391]: cycle detected when 
   |
note: ...which requires ...
  --> $DIR/cycle_crash.rs:4:20
   |
LL | const FOO: usize = FOO;
   |                    ^^^
   = note: ...which again requires , completing the cycle
   = note: cycle used when 
   = note: see https://rustc-dev-guide.rust-lang.org/overview.html#queries and https://rustc-dev-guide.rust-lang.org/query.html for more information

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0391`.
